dashboard/app/models/unit_group.rb

```diff
@@ -217,7 +217,7 @@ def update_scripts(new_units, alternate_units = nil)
     new_units = new_units.reject(&:empty?)
     new_units_objects = new_units.map {|s| Unit.find_by_name!(s)}
     # we want to delete existing unit group units that aren't in our new list
-    units_to_remove = default_unit_group_units.map(&:script) - new_units_objects
+    units_to_remove = default_unit_group_units.map(&:script) + alternate_unit_group_units.map(&:script) - new_units_objects
```

Comment:
Is something calling `unit_group.update_scripts(...)` during the staging/test/prod deploys which causes the alternate_unit_group_unts to be deleted?

Reply:
Yes, it happens during rake seed. here are some breadcrumbs from that codepath:
* https://github.com/code-dot-org/code-dot-org/blob/2bc4295f0c875d0c2e79ec484fc6b65f42a37749/dashboard/lib/tasks/seed.rake#L576
* https://github.com/code-dot-org/code-dot-org/blob/2bc4295f0c875d0c2e79ec484fc6b65f42a37749/dashboard/lib/tasks/seed.rake#L291-L293
* https://github.com/code-dot-org/code-dot-org/blob/f0685e811e4b217919b9a7977b8fe54d9ebc0af5/dashboard/app/models/unit_group.rb#L113-L116
* https://github.com/code-dot-org/code-dot-org/blob/f0685e811e4b217919b9a7977b8fe54d9ebc0af5/dashboard/app/models/unit_group.rb#L128-L130